Project.toml

```diff
@@ -4,8 +4,8 @@ version = "0.2.0"
 
 [deps]
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
-Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
```

Comment:
Why??? Just rollback changes.

Reply:
When I ran the tests (with ]test), the client gave me a warning that it wasn't in the dependencies, and that I should add it with ]add Statistics. Now, when I try removing it, I receive the following error when running tests:

ERROR: LoadError: LoadError: ArgumentError: Package Evolutionary does not have Statistics in its dependencies:
- If you have Evolutionary checked out for development and have
  added Statistics as a dependency but haven't updated your primary
  environment's manifest file, try `Pkg.resolve()`.
- Otherwise you may need to report an issue with Evolutionary

Hence, I believe it is required.

Reply:
Looking further, I believe the package manager automatically reordered the dependencies alphabetically. I tried manually moving the Statistics dependency above the Random dependency and running ]resolve, and it was automatically moved back down.

Reply:
I added `Statistics` dependency to Project.toml  in v0.2.0. If you use `julia --project` in package directory, the package manager will mess up Project.toml file. You could revert this commit and force push your branch.

Reply:
I don't understand why you would edit Project.toml manually when you can use Pkg to manage it much more easily. Considering that Pkg has automatically reordered the Statistics dependency to put it after Random means that I think you should have them in this order. Otherwise, anyone trying to contribute to this project is going to have to manually work around this when dealing with packages, and there are no tangible benefits to having the dependencies listed in a particular order.

Reply:
There are a lot of thing Pkg isn't able to do, so do not get your hope up about it. You needn't to make any changes to Project.toml unless they related to the added functionality.